docs/versioned-plugins/filters/grok-v3.4.1.asciidoc

```diff
@@ -7,12 +7,12 @@ START - GENERATED VARIABLES, DO NOT EDIT!
 :version: v3.4.1
 :release_date: 2017-05-10
 :changelog_url: https://github.com/logstash-plugins/logstash-filter-grok/blob/v3.4.1/CHANGELOG.md
-:include_path: ../../../logstash/docs/include
+:include_path: ../../../../logstash/docs/include
```

Comment:
The path is wrong here. This problem is caused by bad asciidoc in older versions of the plugin. We can fix it with a simple search/replace in the old files.